x-pack/plugins/code/public/plugin.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { Plugin, CoreSetup, CoreStart } from 'kibana/public';
+import moment from 'moment';
+// import { render, unmountComponentAtNode } from 'react-dom';
+import chrome from 'ui/chrome';
+
+import { APP_TITLE } from '../../../legacy/plugins/code/common/constants';
+// import { HelpMenu } from '../../../legacy/plugins/code/public/components/help_menu';
+
+export class CorePluginBPlugin implements Plugin<CorePluginBPluginSetup, CorePluginBPluginStart> {
+  public setup(core: CoreSetup) {
+    console.log(`## new platform code setup`);
+    core.application.register({
```

Comment:
also how is the order of the app on the left hand side menu bar decided in new platform?

Reply:
based on `order`  value https://github.com/restrry/kibana/blob/109192202c633f180209e5fed0bae600a0d14c32/src/core/public/application/types.ts#L46 
the higher number the lower link position https://github.com/restrry/kibana/blob/109192202c633f180209e5fed0bae600a0d14c32/src/core/public/chrome/nav_controls/nav_controls_service.test.ts#L41-L56